read_ext.py

```
@@ -0,0 +1,4 @@
+import sys
+
+with open(sys.argv[1], 'r') as f:
```

Ruff (0.15.12)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)